src/main.py

```diff
@@ -2,12 +2,46 @@
 # -*- coding: utf-8 -*-
 import json
 import os
+import logging
+import collections
 
 import boto3
 import requests
 
 codecommit = boto3.client("codecommit")
-slack_webhook = os.environ["SLACK_WEBHOOK"]
+SLACK_WEBHOOK = os.environ.get("SLACK_WEBHOOK")
+DRYRUN = os.environ.get("DRYRUN")
```

Comment:
Just checking this was tested when passing `dry_run = true` to the terraform module? I think there's a funny thing about environment variables... They're always read as strings... and `if 'false'` would be truthy...

Reply:
```suggestion
DRYRUN = os.environ.get("DRYRUN", "").lower() == 'true'
```

Reply:
updated